packages/app-desktop/main.scss

```diff
@@ -53,7 +53,19 @@ a {
 }
 
 ::-webkit-scrollbar-thumb:hover {
-	background: rgba(100, 100, 100, 0.7);
+	background: var(--scrollbar-thumb-color-hover);
+}
+
+:root {
+	--scrollbar-thumb-color: var(--joplin-scrollbar-thumb-color);
+	--scrollbar-thumb-color-hover: var(--joplin-scrollbar-thumb-color-hover);
+}
+
+// Uses a scrollbar with secondary colors. Should be used for content with
+// background matching joplin-background-color2.
+._scrollbar2 {
```

Comment:
`_scrollbar2` follows the naming pattern for [RSCSS helper classes](https://ricostacruz.com/rscss/helpers.html). RSCSS also suggests including all helpers in a single file called `helpers`. For now, `_scrollbar2` is included in `main.scss` (near the other scrollbar logic). If desired, I can move it to a new `helpers.scss` file.